dashboard/app/views/pd/workshop_enrollment/missing_application.html.haml

```diff
@@ -0,0 +1,9 @@
+%h1
+  Teacher Application Is Required
+
+%p
+  You must have a completed application in order to enroll in this workshop. If
```

Comment:
Nit: It looks like the application must be not only complete, but also accepted and with an RP. Does the user "need" to know those details?

Reply:
This is my bad. We should update the text here to say "You must have an accepted application...."

Reply:
wording has been updated to mention "accepted". Meg, when you say "with an RP" are you referring to the workshop being associated with an RP? I don't think we need to clarify why the application is required, only what state the application needs to be in to satisfy the requirement.